ARK Server Manager/PlayerProfile.xaml.cs

```diff
@@ -0,0 +1,158 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Windows;
+using System.Windows.Documents;
+using System.Windows.Navigation;
+using ARK_Server_Manager.Lib.ViewModel.RCON;
+using ARK_Server_Manager.Properties;
+using ArkData;
+
+namespace ARK_Server_Manager
+{
+    /// <summary>
+    /// Interaction logic for PlayerProfile.xaml
+    /// </summary>
+    public partial class PlayerProfile : Window
+    {
+        public PlayerProfile(PlayerInfo player, String serverFolder)
+        {
+            InitializeComponent();
+
+            this.Player = player;
+            this.ServerFolder = serverFolder;
+            this.DataContext = this;
+        }
+
+        public PlayerInfo Player
+        {
+            get;
+            private set;
+        }
+
+        public Player ArkDataPlayer
```

Comment:
BTW< as an FYI (you don't have to change this), C#6 supports the following lambda-style syntax for read-only properties, which you may find more appealing (and takes up less vertical space.)

public Player ArkDataPlayer => Player?.ArkData;